<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QWidget" name="navigationFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QPushButton" name="dashboardButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>129</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Dashboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="expenseCategoriesButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>129</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Expense Categories</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
